how-to/index.rst

```diff
@@ -41,6 +41,7 @@ Optional Features
    features/ubuntu-pro
    features/validation
    features/vault
+   features/maintenance
```

Comment:
I thunk maintenance should be part of operations instead of features

Reply:
Updated